rpc_shutdown/README.md

```diff
@@ -0,0 +1,77 @@
+# Hass.io Core Add-on: RPC Shutdown
+
+Simple way for remote windows shutdown.
+
+![Supports aarch64 Architecture][aarch64-shield] ![Supports amd64 Architecture][amd64-shield] ![Supports armhf Architecture][armhf-shield] ![Supports armv7 Architecture][armv7-shield] ![Supports i386 Architecture][i386-shield]
+
+## About
+
+Allows you to shut down and Windows Computer with a service call from Home Assistant.
+
+## Installation
+
+The installation of this add-on is straightforward and easy to do.
+
+1. Navigate in your Home Assistant frontend to **Hass.io** -> **Add-on Store**.
+2. Find the "RPC Shutdown" add-on and click it.
+3. Click on the "INSTALL" button.
+
+## How to use
+
+In the configuration section define alias,address and credentials and save the configuration.
+
+1. Start the add-on.
+2. Have some patience and wait a couple of minutes.
+3. Check the add-on log output to see the result.
+
+## Configuration
+
+Add-on configuration:
+
+```json
+{
+  "computers": [
+    {
+      "alias": "test-pc-1",
+      "address": "192.168.0.1",
+      "credentials": "user%password"
+    },
+    {
+      "alias": "test-pc-2",
+      "address": "192.168.0.2",
+      "credentials": "user%password"
+    }
+  ]
+}
+```
+
```

Comment:
Could you please document the individual options below here? To tell people what they are and what to expect?

Reply:
Yes!